p5js/YOLO/sketch.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2018 ml5
+// 
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ML5 Example
+YOLO
+Real time Object Detection
+=== */
+
+let video;
+let yolo;
+let objects = [];
+
+function setup() {
+  createCanvas(640, 410);
+  video = createCapture(VIDEO, onVideoLoaded);
+  video.hide();
+  select('#start').mousePressed(function() {
```

Comment:
Maybe just pass in detect directly? 

```javascript
select('#start').mousePressed(detect);
```

Reply:
ok, I'm still having some issues with the speed (FPS). Although the draw in not waiting for yolo to send results. We can see this tomorrow.